keyboards/4pplet/waffling60/rev_a/rev_a.c

```diff
@@ -0,0 +1,94 @@
+#include "rev_a.h"
+
+void keyboard_pre_init_user(void) {
+  // Call the keyboard pre init code.
+  init_led();
+}
```

Comment:
All the `_user` functions in this file should be `_kb` functions, and call the user functions, so a user may override the behaviour you set here, e.g.:

```suggestion
void keyboard_pre_init_kb(void) {
  // Call the keyboard pre init code.
  init_led();
  keyboard_pre_init_user();
}
```

Reply:
I simply renamed keyboard_pre_init_user to keyboard_pre_init_kb, that sould be the only function affected, right? 😄 led_update_kb calls led_update_user.